lib/models/components-definition.ts

```diff
@@ -148,6 +148,7 @@ export interface ComponentProperty<ControlType = ComponentPropertyControl> {
         | 'styles'
         | 'inlineStyles'
         | 'data'
+        | 'object'
```

Comment:
This is obviously correct but also confusing (we should have never used the name 'object' in Studio/Enterprise). Something like `studio-object` might also be ugly though.

Reply:
You might be right that `studio-object` communicates it more clearly. What do others think?

Reply:
I think `studio-object` makes sense for this case. As long as it isn't reused if the object-select control was sourcing 'objects' from somewhere other than studio

Reply:
Make sense to name it `studio-object`. The accepted values of the properties in the control are also quite linked to Studio at the moment.